File: Indie Effects Git/Assets/IndieEffects/JS Classes/DepthOfField.js
#pragma strict
import IndieEffects;

@script RequireComponent(IndieEffects)
@script AddComponentMenu("Indie Effects/Depth of Field")
var fxRes : IndieEffects;

var shader : Shader;
private var DOFMat : Material;
@range (0,10)
var FStop : float;
@range (0,5)
var BlurAmount : float;

function Start () {
	fxRes = GetComponent(IndieEffects);
	DOFMat = new Material(shader);
}

function OnPostRender () {
	DOFMat.SetTexture("_MainTex",fxRes.RT);
	DOFMat.SetTexture("_Depth",fxRes.DNBuffer);
	DOFMat.SetFloat ("_FStop", FStop*10);
	DOFMat.SetFloat ("_Amount", BlurAmount);
	FullScreenQuad(DOFMat);
}

File: Indie Effects Git/Assets/IndieEffects/CSharp Classes/DepthOfField.cs
using UnityEngine;

[RequireComponent(typeof(IndieEffects))]
[AddComponentMenu("Indie Effects/Depth of Field C#")]
public class DepthOfField : MonoBehaviour
{
    public IndieEffects fxRes;

    public Shader shader;
    private Material DOFMat;
    [Range (0,10)]
    public float FStop;
    [Range (0,5)]
    public float BlurAmount;

    public void Start () {
	    fxRes = GetComponent<IndieEffects>();
	    DOFMat = new Material(shader);
    }

    public void OnPostRender () {
	    DOFMat.SetTexture("_MainTex",fxRes.RT);
	    DOFMat.SetTexture("_Depth",fxRes.DNBuffer);
	    DOFMat.SetFloat ("_FStop", FStop*10);
	    DOFMat.SetFloat ("_Amount", BlurAmount);
	    IndieEffects.FullScreenQuad(DOFMat);
    }
}